integration_tests/src/daphne.rs

```diff
@@ -108,6 +108,53 @@ impl<'a> Daphne<'a> {
             );
         }
 
+        // Reset Daphne's state.
```

Comment:
Unless I'm reading the diff wrong, it looks like the `// Reset Daphne's state` and `// Add an HPKE receiver config to Daphne` blocks are duplicated above this line. i.e. it looks like we're already initializing, or at least attempting to.

Reply:
😮‍💨 oh gosh, how dumb. Fixed.